build/Dockerfile.ci

```diff
@@ -33,6 +33,18 @@ RUN git clone --single-branch --branch release-4.6 https://github.com/openshift/
 WORKDIR ovn-kubernetes/go-controller/
 RUN make windows
 
+# Build promu utility tool
+WORKDIR /build/
```

Comment:
Add some comments on why promu is needed?

Reply:
Sure will add comments. Basically it's a tool to build `windows_exporter.exe` file and the windows_exporter documentation points to using it. I was not able to build the executable from source without this (though if we figure out how to do that we could drop this dependency in the future).